interface/src/Application.cpp

```diff
@@ -3188,12 +3188,18 @@ void Application::initializeUi() {
             // END PULL SAFEURLS FROM INTERFACE.JSON Settings
 
             bool isInWhitelist = false;  // assume unsafe
-            for (const auto& str : safeURLS) {
-                if (!str.isEmpty() && str.endsWith(".qml") && url.toString().endsWith(".qml") &&
-                    url.toString().startsWith(str)) {
-                    qCDebug(interfaceapp) << "Found matching url!" << url.host();
-                    isInWhitelist = true;
-                    return true;
+            
+            if (AUTHORIZED_EXTERNAL_QML_SOURCE.isParentOf(url)) {
+                isInWhitelist = true;
+                return true;
+            } else {
+                for (const auto& str : safeURLS) {
+                    if (!str.isEmpty() && str.endsWith(".qml") && url.toString().endsWith(".qml") &&
+                        url.toString().startsWith(str)) {
+                        qCDebug(interfaceapp) << "Found matching url!" << url.host();
+                        isInWhitelist = true;
```

Comment:
This variable appears to be set but otherwise unused.

Reply:
This is true, I will remove it, can always add it back if we need it.